java-rest-service/accelerator.yaml

```diff
@@ -143,11 +143,10 @@ engine:
           exclude: [ "pom.xml", "build.gradle.kts", "**/application.properties" ]
         - include: [ "pom.xml" ]
           chain:
-            - type: OpenRewriteRecipe
-              recipe: org.openrewrite.maven.RemoveDependency
-              options:
-                groupId: "'org.liquibase'"
-                artifactId: "'liquibase-core'"
+            - type: ReplaceText
+              regex:
+                pattern: "<dependency>\\s+<groupId>org\\.liquibase</groupId>\\s+<artifactId>liquibase-core</artifactId>\\s+</dependency>\\s+"
```

Comment:
Could use single quoted string (avoids need of escaping the backslash) and `\s*` instead of `\s+`

Reply:
Using single quotes only at this place would make it inconsistent with all other places. Changing it everywhere would cause significant testing effort.
I don't see any value of changing \s* to \s+ in this particular case. We know how pom.xml is written and we know that there are spaces and new line characters there. This insignificant change just introduce unnecessary extra effort.

@ericbottard Please merge this PR as is.

Reply:
Fair point